contracts/Registry/TokenCuratedRegistry.sol

```diff
@@ -84,17 +84,8 @@ contract TokenCuratedRegistry is StakedRegistry, TimelockableItemRegistry {
   // Reverts if a challenge for this item id does not exist.
   // Reverts if the challenge has not ended.
   function challengePassed(bytes32 id) public view returns (bool) {
```

Comment:
since this doesn't need additional logic anymore, maybe just get rid of this function and replace with `challenges[id].passed()`